### PR TITLE
Feature/13990 UI automation for pb is13020and13022

### PR DESCRIPTION
**Pull request by kishorambare**
Hi All,

This PR is consist of UI automation of PBIs- 13020 & 13022.
kindly review and approve.
Thanks.

### CODE REVIEW

File: appSetting.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {
     "autoTestConfig":
     {
-        "url":"",
-        "user":"",
-        "password":"",
-        "userFullName" : "",
-        "apiurl": ""
+        "url":"https://fss-dev.admiralty.co.uk",
```
Comment (crdandoukho) [inline appSetting.json:4]: Please revert the changes to autoTestConfig

File: playwright.config.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { autoTestConfig } from './appSetting.json';
 
 const config: PlaywrightTestConfig = {
   
-  retries: 2,
+  retries: 3,
```
Comment (crdandoukho) [inline playwright.config.ts:7]: Please check if these changes to playwright config should be included

**Review by crdandoukho — CHANGES_REQUESTED** [review]

File: appSetting.json (hunk 1 above)
Comment (kishorambare) [reply appSetting.json]: Reverted the changes, Thanks Chris.

File: playwright.config.ts (hunk 2 above)
Comment (kishorambare) [reply playwright.config.ts]: Reverted the changes, Thanks Chris.

File: Tests/FunctionalTests/ess-ui-encselectionpage.spec.ts [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+      await expect(page.locator(esslandingpageObjectsConfig.exchangesettextSelector)).toBeVisible();
+   })
+
+   // https://dev.azure.com/ukhocustomer/File-Share-Service/_workitems/edit/13954 - Add Anther ENC
+   // https://dev.azure.com/ukhocustomer/File-Share-Service/_workitems/edit/13956 - Duplicate ENC
+   test('Verify that after clicking on "Add another ENC" link, user able to add another ENC number', async ({ page }) => {
+      await addSingleENC(page, esslandingpageObjectsConfig.addSingleENCTextboxSelector);
+      await expect(page.locator(encselectionpageObjectsConfig.addAnotherENCSelector)).toBeVisible();
+      await addAnotherENC(page, encselectionpageObjectsConfig.addAnotherENCSelector);
+      expect(await page.isChecked(encselectionpageObjectsConfig.chooseBoxSelecetor)).toBeFalsy();
```
Comment (RaviK2233) [inline Tests/FunctionalTests/ess-ui-encselectionpage.spec.ts:37]: Test also need to verify that the ENC cell added in the grid is same as ENC entered in text box.

File: Tests/FunctionalTests/ess-ui-encselectionpage.spec.ts [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+      //13956 - Add another ENC2 - Duplicate No.
+      await addAnotherENC(page, encselectionpageObjectsConfig.addAnotherENCSelector);
+      await expect(page.locator(encselectionpageObjectsConfig.errorMessageForDuplicateNumberSelector)).toContainText(encselectionpageObjectsConfig.errorMsgDuplicateENC)
```
Comment (RaviK2233) [inline Tests/FunctionalTests/ess-ui-encselectionpage.spec.ts:40]: Test also need to verify that Duplicate ENC is not added in the Grid.

**Review by RaviK2233 — CHANGES_REQUESTED** [review]
Please see comment for test 13954/13956

File: Tests/FunctionalTests/ess-ui-encselectionpage.spec.ts (hunk 4 above)
Comment (kishorambare) [reply Tests/FunctionalTests/ess-ui-encselectionpage.spec.ts:41]: Hi Ravi,
Added this check.
Thanks.

File: Tests/FunctionalTests/ess-ui-encselectionpage.spec.ts (hunk 3 above)
Comment (kishorambare) [reply Tests/FunctionalTests/ess-ui-encselectionpage.spec.ts:38]: Hi Ravi,
Added this check.
Thanks.

**Review by crdandoukho — APPROVED** [review]
Thanks Kishor for your changes. I am happy to merge but will await approval from Ravi

**Review by RaviK2233 — APPROVED** [review]
